src/components/graph/graph.ts

```diff
@@ -350,17 +367,77 @@ export default class GraphComponent {
    * @private
    * @return {{array, number}} A tuple of the x, y translation and the zoom scale
    */
-  private _getGraphTranslationAndScale(): {translation: number[], scale: number} {
+  private _getGraphTranslationAndScale(): { translation: number[], scale: number } {
     const transformRaw = this._g.attr('transform')
-    if (transformRaw === null) return {translation: [0, 0], scale: 1}
+    if (transformRaw === null) return { translation: [0, 0], scale: 1 }
     const [translationRaw, scaleRaw] = transformRaw.split(' ')
     const translation = translationRaw
       .replace('translate(', '')
       .replace(')', '')
       .split(',')
       .map(Number)
     const scale = Number(scaleRaw.match(/\d(\.\d+)*/)[0])
-    return {translation, scale}
+    return { translation, scale }
+  }
+
+  /**
+   * @private
+   * @return {{array}} A tuple of the x, y position relative to the SVG
```

Comment:
drop the doc string. We'll figure it all out later. it's kinda useless with the typescript type annotations anyway